### ISSUE TITLE
dropTable auto quoting fails with Oracle

### ISSUE THREAD

**Issue by jonny7** [yiisoft/yii2#10298]
I am using Command.php to do some DB updates and I find that the automatic quoting of the table name for Oracle 11g means that the query doesn't work. I have never really used this before so I am wondering if there is an accepted work around here?

    try {
				Yii::$app->db->createCommand()->dropTable($r->QUERY)->execute();
				$actions['success'][] = $r->QUERY . ' successfully dropped';
    }
The query that fails is ` SQLSTATE[HY000]: General error: 942 OCIStmtExecute: ORA-00942: table or view does not exist (ext\pdo_oci\oci_statement.c:148) The SQL being executed was: DROP TABLE "my_table"`

**Comment by samdark** [comment]
What's intended SQL to be generated?

**Comment by jonny7** [comment]
I found that if I added `Yii::$app->db->createCommand()->dropTable(strtoupper($r->QUERY))->execute();`

Then the command executed fine.

**Comment by samdark** [comment]
Looks like something like https://github.com/yiisoft/yii2/issues/9457

**Comment by jonny7** [comment]
Yes definitely around the same topics. I personally always use MySQL/Maria. This is the first time I've used Oracle with Yii.
